CHANGELOG

```diff
@@ -1,12 +1,27 @@
 devel
 -----
 
+* Don't store selectivity estimate values for newly created system collections.
+
+  Not storing the estimates has a benefit especially for the `_statistics`
+  system collections, which are written to periodically even on otherwise
+  idle servers. In this particular case, the actual statistics data was way
+  smaller than the writes caused by the index estimate values, causing a
+  disproportional overhead just for maintaining the selectivity estimates.
+  The change now turns off the selectivity estimates for indexes in all newly 
+  created system collections, and for new user-defined indexes of type
+  "persistent", "hash" or "skiplist", there is now an attribute "estimates"
+  which can be set to `true` to disable the selectivity estimates for the index.
```

Comment:
The attribute is just called "estimates" and a value of "true" _disables_ the estimates? I would find that very counter intuitive! I suppose this should be:
```suggestion
  which can be set to `false` to disable the selectivity estimates for the index.
```